corehq/ex-submodules/pillowtop/es_utils.py

```diff
@@ -79,6 +79,13 @@ def _get_analysis(*names):
             "analysis": _get_analysis('default'),
         },
     },
+    "production": {
+        CASE_SEARCH_HQ_INDEX_NAME: {
+            "settings": {
+                "number_of_shards": 50,
+            },
+        },
+    }
```

Comment:
Could this be done in commcare-cloud `production/elasticsearch.yml` instead? Or is there something that can be only done here?

See https://github.com/dimagi/commcare-cloud/blob/da13063249289b7208bb4e1a7bd8bde1362e618f/environments/staging/elasticsearch.yml#L3 as an example of this file. It's used to populate the `ES_SETTINGS` variable in `localsettings.py`

Reply:
It can be done in commcare-cloud but the process is bit trickier since we want this present in the limited release but not in the current release. Perhaps it doesn't matter since I don't think it get's used outside of creating the index and perhaps a few management commands.

If this were tied to the actual index name I would feel a bit safer since then it would only apply to the new index and not the old one.